router/internal/epoller/epoll_linux.go

```diff
@@ -99,7 +103,7 @@ func (e *Epoll) Wait(count int) ([]net.Conn, error) {
 	events := make([]unix.EpollEvent, count)
 
 retry:
-	n, err := unix.EpollWait(e.fd, events, -1)
+	n, err := unix.EpollWait(e.fd, events, e.timeoutMsec)
```

Comment:
Did you find out why -1 is used on linux? Without any good argument I would not change the default.

Reply:
-1 = no timeout